### DISCUSSION TITLE
Live OpenSSL Providers Workshop: Authors Track : Dec 11th and Dec 12th

### DISCUSSION THREAD

**Discussion by sapkkaj** [openssl/openssl#22946] [Announcements]
Part two of the OpenSSL Providers Workshop is next week! We have divided the workshop into two tracks the Users Track and the Authors Track. Please join us next week for part two of the workshop: Live OpenSSL Providers Workshop: Authors Track. As with the Users Track, we will be hosting two sessions of the Users Track at different times to allow people from different time zones to be able to join our workshops live.

The Author Track will cover how to write your own OpenSSL provider. This session will assume some basic knowledge about what OpenSSL providers are and how to use them (such as might be obtained from attending the “Users Track” session). It will be split into 4 separate presentations by OpenSSL Engineers. There will be opportunities to ask questions after each talk, as well as at the end where there will be an open forum for any questions or feedback not covered by the individual presentations.

Learn more and register in advance for the workshop here(please choose the time zone that works best for you):

Session 1: Americas and EMEA Time Zone

When: Dec 11, 2023 04:00 PM Universal Time UTC
Register in advance for this webinar:
https://zoom.us/webinar/register/WN_2UqTPnrxQjyUJOzUxOj77w

Session 2: APAC Time Zone

When: Dec 12, 2023 07:00 AM Universal Time UTC
Register in advance for this webinar:
https://zoom.us/webinar/register/WN_LNFArIEmQmqbmiLdSuOdOA

After registering, you will receive a confirmation email containing information about joining the webinar.

Contact us if you have any questions or comments

**Comment by rsbeckerca** [comment]
Question on the above: is it possible to get the recordings/slides from the presentations yesterday?

**Comment by mattcaswell** [comment]
Yes. Email feedback@openssl.org to request the slides. Recordings will be on our Youtube channel at some point on Wednesday:

[https://www.youtube.com/@openssl_](https://www.youtube.com/@openssl_)

**Reply by gingerbeard-man** [reply]
FYI it seems GitHub(?) has truncated the hyperlink.  The trailing underscore is part of the channel name.

[https://www.youtube.com/@openssl_](https://www.youtube.com/@openssl_)

**Reply by mattcaswell** [reply]
Should be fixed now
